pkg/config/v2/config.go

```diff
@@ -189,6 +189,20 @@ func (c cfg) GetResources() ([]resource.Resource, error) {
 	return resources, nil
 }
 
+// ListResourcesWithGK implements config.Config
+func (c cfg) ListResourcesWithGK(gvk resource.GVK) []resource.Resource {
```

Comment:
wondering if we need these changes in v2 plugin, since we will be deprecating it soon.

Reply:
We should not change the v2 +1 Good Point.

Reply:
I understand.